### DISCUSSION TITLE
How are you coloring results in a file paths search?

### DISCUSSION THREAD

**Discussion by olimorris** [PatrickF1/fzf.fish#259] [Q&A] [answered]
In my latest [dotfiles commit](https://github.com/olimorris/dotfiles/commit/c0a7b5d0fe9dc770818b41da291fbaa0446307b7), I have a really nice way of updating the light and dark versions of my fish instance when using `fzf.fish` by updating `FZF_DEFAULT_OPTS `:

<img width="1503" alt="Screen Shot 2022-08-31 at 22 49 31@2x" src="https://user-images.githubusercontent.com/9512444/187791374-befd39bf-97b1-4dfc-89bc-e91da30b0ab1.png">

However, I can't workout how to get a file paths search to look the same:

<img width="1502" alt="Screen Shot 2022-08-31 at 22 49 49@2x" src="https://user-images.githubusercontent.com/9512444/187791426-8126b9a7-f63a-4cbe-bc50-bebdbd76eac2.png">

Is altering `LSCOLORS` the only way?

**Comment by PatrickF1** [comment, marked as answer]
The files path search is automatically colored by fd. https://github.com/PatrickF1/fzf.fish/blob/main/functions/_fzf_search_directory.fish#L3

Edit: Looks like you do have to modify LS_COLORS to control it. https://github.com/sharkdp/fd#colorized-output
